pkg/networkpath/traceroute/runner/runner.go

```diff
@@ -149,10 +151,18 @@ func (r *Runner) RunTraceroute(ctx context.Context, cfg config.Config) (payload.
 		}
 	case payload.ProtocolUDP:
 		log.Tracef("Running UDP traceroute for: %+v", cfg)
-		pathResult, err = r.runUDP(cfg, hname, dest, maxTTL, timeout)
-		if err != nil {
-			tracerouteRunnerTelemetry.failedRuns.Inc()
-			return payload.NetworkPath{}, err
+		if runtime.GOOS == "windows" {
```

Comment:
I would have expected two versions of `runUDP` instead, each in its own os-specific file.

Reply:
Moved this out so the Windows UDP and Dublin UDP has separate OS bound files